lib/tests/streamlit/elements/checkbox_test.py

```diff
@@ -99,3 +105,27 @@ def test_checkbox_help_dedents(self):
         self.assertEqual(c.label, "Checkbox label")
         self.assertEqual(c.default, True)
         self.assertEqual(c.help, "hello\n world\n")
+
+    @parameterized.expand(
+        [
+            ("visible", LabelVisibilityMessage.LabelVisibilityOptions.VISIBLE),
+            ("hidden", LabelVisibilityMessage.LabelVisibilityOptions.HIDDEN),
+            ("collapsed", LabelVisibilityMessage.LabelVisibilityOptions.COLLAPSED),
+        ]
+    )
+    def test_label_visibility(self, label_visibility_value, proto_value):
+        """Test that it can be called with label_visibility param."""
+        st.checkbox("the label", label_visibility=label_visibility_value)
+
+        c = self.get_delta_from_queue().new_element.checkbox
+        self.assertEqual(c.label, "the label")
+        self.assertEqual(c.label_visibility.value, proto_value)
+
```

Comment:
Would it be possible to write a test for the 3rd scenario where `label=""` and the warning is raised?

Reply:
Tests added!